### PR TITLE
Add bépo layout and several diacritics support

### PR DESCRIPTION
**Pull request by Phundrak**
With these commits eDEX-UI should now support the bépo layout, several new diacritics (acute and grave accents, the caron, bar, breve, macron, cedilla, overring, iota subscript diacritics), and it should also support the Greek letters when used with the bépo dead key `AltGr+g`.

Also, since the bépo layout makes often use of Shift+AltGr+key characters, maybe adding a combination of `alt_cmd` and `shift_cmd` could be considered?

For reference, the bépo layout as it is currently declared:
2018-11-21-233326_4480x1080_scrot

And how the full layout would be organized if Shift+AltGr were available:
carte-bepo-complete

### CODE REVIEW

File: src/classes/keyboard.class.js [hunk 1]
```diff
@@ -103,6 +103,27 @@ class Keyboard {
                     return "ĉ";
                 case "C":
                     return "Ĉ";
+                // the circumflex can also be used for subscript numbers
```
Comment (GitSquared) [inline src/classes/keyboard.class.js]: I believe you meant:
```suggestion
                // the circumflex can also be used for superscript numbers
```

File: src/classes/keyboard.class.js [hunk 2]
```diff
@@ -152,6 +173,603 @@ class Keyboard {
...
+                return "Ů";
+            case "w":
+                return "ẘ"; // capital w with overring not supported on bépo layout apparently
+            case "y":
+                return "ẙ"; // same for capital y with overring
+            default:
+                return char;
+            }
+        };
+        let toGreek = (char) => {
```
Comment (GitSquared) [inline src/classes/keyboard.class.js]: I don't believe adding a modifier is the right way to handle different alphabets. You should define them directly in the keyboard file, in `*_cmd` fields. See my review on the BEPO layout file

File: src/assets/kb_layouts/fr-BEPO.json [hunk 3]
```diff
@@ -0,0 +1,490 @@
+{
```
Comment (GitSquared) [inline src/assets/kb_layouts/fr-BEPO.json]: BEPO layout is all good.

**Comment by GitSquared** [comment]
Also, do you think using `capslck_` instead of a shift+alt combination would be a viable option? It's already used on the french keyboard to type things like "É".

File: src/classes/keyboard.class.js (hunk 1 above)
Comment (Phundrak) [reply src/classes/keyboard.class.js]: Ah yes indeed, I might have wanted to write “supscript” instead at that time

**Comment by Phundrak** [comment]
> Also, do you think using `capslck_` instead of a shift+alt combination would be a viable option? It's already used on the french keyboard to type things like "É".

I don’t think so, generally using the Capslock already has a defined behavior for a few keys (which I just realized I forgot to include, new commit coming soon). Plus I think it is more intuitive for bépo users to type AltGr+Shift+o to get the “Œ” char when they have to use the AltGr+o combination to get “œ”. Maybe it could be used as an alternative, but I think it would be best to have AltGr+Shift available.

**Comment by GitSquared** [comment]
> 
> 
> > Also, do you think using `capslck_` instead of a shift+alt combination would be a viable option? It's already used on the french keyboard to type things like "É".
> 
> I don’t think so, generally using the Capslock already has a defined behavior for a few keys (which I just realized I forgot to include, new commit coming soon). Plus I think it is more intuitive for bépo users to type AltGr+Shift+o to get the “Œ” char when they have to use the AltGr+o combination to get “œ”. Maybe it could be used as an alternative, but I think it would be best to have AltGr+Shift available.

All right, just wanted to make sure. I'll make a alt+shift prefix.

**Comment by Phundrak** [comment]
Great! I also just added the needed lines for the capslock behavior I was talking about earlier.

**Comment by GitSquared** [comment]
@Phundrak `altshift` prefix support is up.

**Comment by GitSquared** [comment]
@Phundrak are still planning to add `altshift` support on this keyboard or should i merge this PR already?

**Comment by Phundrak** [comment]
I should add the `altshift` support during this week-end, so maybe keep it open for now

**Comment by GitSquared** [comment]
@Phundrak I'm making a new release so i'm merging this; see the issue on #289.

**Pull request merged**